pgjdbc/src/main/java/org/postgresql/core/PGStream.java

```diff
@@ -52,6 +52,7 @@ public class PGStream implements Closeable, Flushable {
   private Socket connection;
   private VisibleBufferedInputStream pgInput;
   private PgBufferedOutputStream pgOutput;
+  private int protocol;
```

Comment:
WDYT of creating `class ProtocolVersion { int major; int minor; }`? (or `short major, minor`)